BHoM_UI/Components/Engine/GetInfo.cs

```diff
@@ -60,11 +63,58 @@ public class GetInfoCaller : MethodCaller
+        [MultiOutput(0, "description", "Introduction to what the BHoM is all about.")]
+        [MultiOutput(1, "version", "Version of the BHoM currently running in this interface.")]
+        [MultiOutput(2, "installer", "If you have used the BHoM installer, this gives you information about its exact version.")]
+        [MultiOutput(3, "toolkits", "List of toolkits available in this version of the BHoM.")]
+        [MultiOutput(4, "types", "List of objects types available in this version of the BHoM.")]
+        [MultiOutput(5, "methods", "List of methods available in the version of the BHoM.")]
+        [MultiOutput(6, "adapters", "List of adapters available in the version of the BHoM.")]
+        [MultiOutput(7, "website", "BHoM website.")]
+        [MultiOutput(8, "github", "GitHub organisation containing all our open-source toolkits.")]
+        [MultiOutput(9, "wiki", "Need more details on how to use the BHoM or how to contribute? This is the place to go.")]
+        public static Output<string, string, string, List<string>, List<Type>, List<MethodInfo>, List<Type>, string, string, string> GetInfo()
         {
-            return "This is the BHoM";
+            string description = "This is the BHoM (Buildings and Habitats object Model), a collaborative computational development project for the built environment. "
+                + "It is a collective effort to share code and standardise the data that we use to design, everyday � across all activities and all disciplines.\n\n"
+                + "It is not an attempt to standardise exact processes � these must be flexible...\n\n"
+                + "It is also not an attempt to standardise the software we use...\n\n"
+                + "But in standardising the data but not the data-base, we provide great opportunities for efficiencies, for collaboration and most of all, to change the way we work.\n\n"
+                + "It is crafted as transdisciplinary, software-agnostic and office/region/country-invariant, and therefore would be nothing without our active community and wide range of contributors.\n\n"
+                + "The whole BHoM project uses an open-source model for project architecture, co-creation and planning. So explore, experiment and contribute to both the source code and the wiki. "
+                + "Sharing and building our code together in this open-source type approach means we can feed off and pool our disparate knowledge, experience and expertise towards a common goal � better design.\n\n"
+                + "Also, as well as creating a common language of BHoM objects - the large number of repositories contain a variety of different plugins and code to operate on BHoM objects and link the BHoM with our favourite software and tools. "
+                + "Much of the core code is written in C#. But we also have code in JavaScript, C++, Python and visual programming languages such as Grasshopper User Objects and Dynamo Custom Nodes all forming part of the BHoM.";
```

Comment:
Hey - thought it would be good to make this quite a bit shorter. Means we don't have to have lots of duplicated text in the code to worry about revising and keeping up to date. As that prose from the wiki could indeed probably do with a refresh anyway. 
Repeated the link to the website - although like the explicit website output too!